RussianFilter/sections/specific.txt

```diff
@@ -18,6 +18,7 @@ Ly9hbi55YW5kZXgucnUv
 ! adbetnet domains
 ||cdn.*/libs/b.js$third-party,domain=nnm-club.lib|nnm-club.me|nnm-club.tv|nnmclub.to|seasonvar.ru|mintmanga.com|realmadrid.ru
 !
+ccleaner.com##.trustpilot-widget
```

Comment:
Not an ad

Reply:
it is a third-party widget and now it's blocking by Tracking Protection. so maybe we should unblock it in Spyware filter 
`@@||googletagmanager.com/gtm.js$domain=ccleaner.com`
and block `##.trustpilot-widget` by Annoyance filter

Reply:
@slavaleleka and `||widget.trustpilot.com^$third-party`